webapp/src/components/link_tooltip/link_tooltip.test.jsx

```diff
@@ -1,43 +1,67 @@
 import {getInfoAboutLink} from './link_tooltip';
 
 describe('getInfoAboutLink should work as expected', () => {
-    it('Should return the correct owner, repo, type, and number when given a valid GitLab href and hostname with all required information', () => {
+    it('Should return an object of correct owner, repo, type, and number when given a valid GitLab hostname and href with all required information', () => {
         const href = 'https://gitlab.com/mattermost/test/-/merge_requests/123';
         const hostname = 'gitlab.com';
-        const expected = ['mattermost', 'test', '-', 'merge_requests', '123'];
+        const expected = {
+            owner: 'mattermost',
+            repo: 'test',
+            type: 'merge_requests',
+            number: '123',
+        };
 
         const result = getInfoAboutLink(href, hostname);
-
         expect(result).toEqual(expected);
     });
 
-    it('Should return the correct owner, repo, type, and undefined number when given a valid GitLab href and hostname missing the number', () => {
+    it('Should return an object of correct owner, repo, and type when given a valid GitLab hostname and href missing the number', () => {
         const href = 'https://gitlab.com/mattermost/test/-/merge_requests';
         const hostname = 'gitlab.com';
-        const expected = ['mattermost', 'test', '-', 'merge_requests'];
+        const expected = {
+            owner: 'mattermost',
+            repo: 'test',
+            type: 'merge_requests',
+        };
 
         const result = getInfoAboutLink(href, hostname);
-
         expect(result).toEqual(expected);
     });
 
-    it('Should return the correct owner, repo, undefined type, and number when given a valid GitLab href and hostname missing the type', () => {
+    it('Should return the correct owner and repo when given a valid GitLab hostname and href missing the type', () => {
         const href = 'https://gitlab.com/mattermost/test/123';
         const hostname = 'gitlab.com';
-        const expected = ['mattermost', 'test', '123'];
+        const expected = {
+            owner: 'mattermost',
+            repo: 'test',
+        };
 
         const result = getInfoAboutLink(href, hostname);
-
         expect(result).toEqual(expected);
     });
 
-    it('Should return an array of empty strings when given a valid href and empty hostname', () => {
+    it('Should return an object with empty owner when given an empty hostname and valid href', () => {
         const href = 'https://gitlab.com/mattermost/test/-/merge_requests/123';
         const hostname = '';
-        const expected = [''];
+        const expected = {
+            owner: '',
+        };
 
         const result = getInfoAboutLink(href, hostname);
+        expect(result).toEqual(expected);
+    });
+
+    it('Should return an object invalid owner, repo, type, and number when given a valid hostname and invalid href', () => {
```

Comment:
Add a testcase for issue comment as well as mentioned in the [comment](https://github.com/mattermost/mattermost-plugin-gitlab/pull/301#discussion_r1153742764)